frontend/public/components/monitoring/silence-form.tsx

```diff
@@ -37,13 +40,13 @@ const DatetimeTextInput = (props) => {
       <Tooltip
         content={[
           <span className="co-nowrap" key="co-timestamp">
-            {isValid ? new Date(props.value).toISOString() : 'Invalid date / time'}
+            {isValid ? new Date(props.value).toISOString() : t('monitoring~Invalid date / time')}
```

Comment:
I believe this file is also covered by #7386.

Reply:
Yes. it is covered in #7386.